tools/diff-processor/breaking_changes/field_diff.go

```diff
@@ -13,13 +13,15 @@ import (
 // regarding field attribute changes
 type FieldDiffRule struct {
 	Identifier string
-	Messages   func(resource, field string, fieldDiff diff.FieldDiff) []string
+	Messages   func(resource, field string, fieldDiff diff.FieldDiff, schemaDiff diff.SchemaDiffInterface) []string
```

Comment:
to clarify - I didn't mean passing the entire schema diff here, just the resource diff. I don't think there would ever be a case where a field on one resource needs information about a field on a different resource.

Reply:
simplified scope to use minimal interface for needed action